sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -176,3 +176,22 @@ UNIQUE KEY `ip_string` (`ip_string`)
 ) ENGINE=InnoDb AUTO_INCREMENT=1;
 #EndIf
 
+#IfNotTable email_queue
+CREATE TABLE `email_queue` (
+`id` bigint NOT NULL auto_increment,
+`sender` varchar(255) DEFAULT '',
+`recipient` varchar(255) DEFAULT '',
+`subject` varchar(255) DEFAULT '',
+`body` text,
+`sent` tinyint DEFAULT 0,
+`error` tinyint DEFAULT 0,
+`error_message` text,
+PRIMARY KEY (`id`),
+KEY `sent` (`sent`)
```

Comment:
for logging purposes, need to add following datetime fields
datetime_queued
datetime_sent